src/main/java/org/folio/inventory/domain/instances/InstanceRelationship.java

```diff
@@ -8,32 +8,36 @@
  */
 public class InstanceRelationship {
   // JSON property names
+  public static final String REALTED_INSTANCE_ID_KEY = "relatedInstanceId";
   public static final String SUPER_INSTANCE_ID_KEY = "superInstanceId";
   public static final String SUB_INSTANCE_ID_KEY = "subInstanceId";
   public static final String INSTANCE_RELATIONSHIP_TYPE_ID_KEY = "instanceRelationshipTypeId";
 
   public final String id;
+  public final String relatedInstanceId;
   public final String superInstanceId;
   public final String subInstanceId;
   public final String instanceRelationshipTypeId;
 
-  public InstanceRelationship (String id, String superInstanceId, String subInstanceId, String instanceRelationshipTypeId) {
+  public InstanceRelationship (String id, String relatedInstanceId, String superInstanceId, String subInstanceId, String instanceRelationshipTypeId) {
```

Comment:
`InstanceRelationship` seemed to reference all relationships which an instance had to another instance. Given that a new vector of relationship was being introduced, it seemed like this class needed to be expanded to include this new relationship. If it was the intent for each relationship to be expressed in its own class, then it seemed like parent and child would be split our into their own classes as well. Given that they were not, I decided to follow that pattern and not give the sibling relationship its own class.

Reply:
> `InstanceRelationship` seemed to reference all relationships which an instance had to another instance. 

It does reference all instance relationships. These are different to related instances. The SIG / PO's have deliberately separated the two (although the names are confusingly similar) and intend to use them for different purposes.

> Given that a new vector of relationship was being introduced, it seemed like this class needed to be expanded to include this new relationship. 

I think we might have a different understanding of what is being introduced.

Instance relationships and related instances are not two variations of the same thing, though I can understand why folks (including me) thought they were.

> If it was the intent for each relationship to be expressed in its own class, then it seemed like parent and child would be split our into their own classes as well. Given that they were not, I decided to follow that pattern and not give the sibling relationship its own class.

The parent and child instance relationships are modelled differently because there is a direction to those relationships.

Whereas, I believe related instances are symmetrical, in that there is no direction or precedence to one end or the other.

Does any of this make sense? I get the sense that by suggesting that related instances are similar to instance relationships I've caused you to think they are two variations of the same thing.